src/client/Microsoft.Identity.Client/Internal/Broker/BrokerSilentRequest.cs

```diff
@@ -83,25 +81,22 @@ internal void CreateRequestParametersForBroker()
 
         internal void ValidateResponseFromBroker(MsalTokenResponse msalTokenResponse)
         {
-            _authenticationRequestParameters.RequestContext.Logger.Info(LogMessages.CheckMsalTokenResponseReturnedFromBroker);
+            _logger.Info(LogMessages.CheckMsalTokenResponseReturnedFromBroker);
             if (msalTokenResponse.AccessToken != null)
             {
-                _authenticationRequestParameters.RequestContext.Logger.Info(
-                    LogMessages.BrokerResponseContainsAccessToken +
-                    msalTokenResponse.AccessToken.Count());
```

Comment:
AccessToken.Count() = the number of chars in an access token. 
Log message looks like this "Token response contains access token count 2035" 
App developer and @bgavrilMS  super confused.